packages/nodes-from-anchor/test/v01/InstructionAccountNode.test.ts

```diff
@@ -115,6 +115,35 @@ test('it flattens nested instruction accounts', () => {
     ]);
 });
 
+test('it prevents duplicate names by prefixing nested accounts with different parent names', () => {
```

Comment:
Thanks for adding tests! Would you mind adding another one that checks at least two layers of nested accounts here as well?